docs/PowerAuth-SDK-for-iOS.md

```diff
@@ -789,6 +789,32 @@ You can remove biometry related factor data used by Touch or Face ID support by
 PowerAuthSDK.sharedInstance().removeBiometryFactor()
 ```
 
+### Fetch biometry credentials in advance
```

Comment:
I would write this documentation in a much more defensive manner, to mention right away that this is not something that should be typically used. We can also use our `<!-- begin box warning -->` macro.